uint32 seq 
tableObjectPos pop_pos
---
uint32 seq
bool pop_result # for each particle, there exists a corresponding result